testsuite/layers-expansion/pom.xml

```diff
@@ -869,6 +870,7 @@
                                                 <layer>messaging-activemq</layer>
                                                 <layer>micrometer</layer>
                                                 <layer>mod_cluster</layer>
+                                                <layer>mvc-krazo</layer>
```

Comment:
Perhaps add a comment that says that being preview, nothing is provisioned although present. I understand that this comment would have to be removed when promoting this subsystem.

Reply:
I think that is the wildfly full all layers for HA.

Reply:
I misread where in the code this line is. +1 -- this is the same as your other comment above and I'll do the same re commenting it out.

Reply:
@jfdenise Done. (I moved the whole thing lower into the section with 'wildfly' FP only layers.)